package.json

```diff
@@ -15,6 +15,7 @@
     "@next/font": "13.1.2",
     "@supabase/supabase-js": "^2.4.0",
     "bullmq": "^3.10.1",
+    "dotenv": "^16.1.4",
```

Comment:
I don't think we need this either. NextJS comes with dotenv.

Reply:
At least Matthew and I didn't need it

https://nextjs.org/docs/pages/building-your-application/configuring/environment-variables